.circleci/config.yml

```diff
@@ -82,8 +82,8 @@ references:
     run:
       name: Install ruby gems
       command: |
-        gem install bundler -v 2.0.2
-        bundle install --without development --path=vendor/bundle --jobs=4 && bundle clean
+        gem install bundler -v 2.2.15
+        bundle install --path=vendor/bundle --jobs=4 && bundle clean
```

Comment:
curious as to why `--without development` is removed?

Reply:
TLDR; it didn't work otherwise! :D

Longer version, for reasons I still don't understand, RSpec was not being installed when we ignored the development tag (possibly because it's in the `:development` and `:test` shared group) 

So, as we don't pass the tag in apply or CFE either, I decided to remove it! 😄